client/src/modules/events/components/Actions.tsx

```diff
@@ -143,16 +146,22 @@ export const Actions = ({
   }
 
   async function onCancelAttendance() {
-    const ok = await confirm({
+    const cancelMessage: {
+      title: string;
+      body?: string;
+    } = {
       title: 'Are you sure you want to cancel your attendance?',
-    });
+    };
+    if (waitlist.length > 0) {
+      cancelMessage.body = 'The next person on the list will take your seat';
+    }
+    const ok = await confirm(cancelMessage);
```

Comment:
```suggestion
    const ok = await confirm({
      title: 'Are you sure you want to cancel your attendance?',
      ...(waitlist.length > 0 &&
        !inviteOnly && {
          body: 'The next person on the waitlist will take your seat.',
        }),
    });
```
Creating `cancelMessage` with optional `body` property, and then conditionally adding it separately is a bit verbose. What do you think about conditionally including it in this way?

Reply:
Agree, cleaner this way